### PR TITLE
TC intensity work

### PR DESCRIPTION
**Pull request by kieranricardo**
@wcarthur this PR contains all my TC intensity work so far - sorry its so long!

### CODE REVIEW

**Comment by wcarthur** [comment]
@kieranricardo did you make any mods to `hurr.f` from the original source? If so, just add a comment at the top of that file noting any changes.

**Comment by kieranricardo** [comment]
hey @wcarthur - I've added some documentation and fixed that bug I was talking about. Can have a look at this code and see whether it makes sense to you how to run/use it?

here's still the issue of too many low intensity storms though, this might be a parameter issue. Also Emanuel tends to initialise storms with 20m/s vortices rather than initial vortices sampled from the historical record so maybe a weakness of this model is that the threshold of intensity required for a vortex to intensify is unrealistically high. Anyway I'll pop this into a word doc

File: intensity/temp_extract.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#PBS -l walltime=1:30:00
+#PBS -l mem=96GB,ncpus=48,jobfs=4000MB
+#PBS -W umask=0002
+#PBS -joe
+#PBS -l storage=gdata/v10+gdata/gb6
+
+module purge
+
+module use /g/data/v10/public/modules/modulefiles
+module load dea/20210527
```
Comment (wcarthur) [inline intensity/temp_extract.sh]: is this interchangeable with 

```
module use /g/data/hh5/public/modules
module load conda/analysis3
```

File: intensity/test_intensity.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+sys.path.append(os.path.expanduser("~/geoscience/repos/tcha"))
```
Comment (wcarthur) [inline intensity/test_intensity.py]: Prefer to append to $PYTHONPATH in the shell rather than in the python code script

**Review by wcarthur — CHANGES_REQUESTED** [review]
@kieranricardo Can you confirm if there's any input data in `~/geoscience/data`  

Can you check if the python modules available through `v10` match with what's available through `hh5`? THese are used in the `temp_extract.sh` scripts. I've swapped the `v10` `dea20210527` with `hh5`'s `conda/analysis3` python environment in other code with no problem

**Comment by kieranricardo** [comment]
@wcarthur all the data is now in /g/data/w85/tc_intensity_data

File: intensity/temp_extract.sh (hunk 1 above)
Comment (kieranricardo) [reply intensity/temp_extract.sh]: I'm not in hh5 but I only us:

- xarray
- gdal
- numpy
- pyproj
- scipy
- mpi4py

**Review by wcarthur — APPROVED** [review]

**Pull request merged**